backstop/tests/information-panel.html

```diff
@@ -139,40 +143,28 @@
         height: 768px;
       }
 
-      .demo-information-table {
-        table-layout: fixed;
-        width: 100%;
-      }
-
-      tr {
-        height: 33px;
+      .demo-usericon-with-label {
+        display: flex;
+        align-items: center;
+        gap: 4px;
       }
 
-      td {
-        white-space: nowrap;
-        overflow: hidden;
-        text-overflow: ellipsis;
+      /* Removes border and padding from readonly text fields */
+      input[readonly] {
```

Comment:
What if we handle that in CSS? Or it might be unwanted styling for the users?

Reply:
Yeah I would like to leave this component to be mostly layout